composer/algorithms/alibi/alibi.py

```diff
@@ -26,15 +23,10 @@
 
 def apply_alibi(
     model: torch.nn.Module,
-    heads_per_layer: int,
-    max_sequence_length: int,
-    position_embedding_attribute: str,
-    attention_module: Type[torch.nn.Module],
-    attr_to_replace: str,
-    alibi_attention: Callable,
-    mask_replacement_function: Optional[Callable[[torch.nn.Module, int], torch.nn.Module]] = None,
+    max_sequence_length: int = 8192,
```

Comment:
8192 was the default when I picked up this code, but I'm glad you bring this up because I'm realizing that we should lower that substantially. The size of the buffers become substantial at that size. If someone wants to use 8192 (or larger), they should do so by choice rather than by default.

Thinking more, perhaps this shouldn't have a default value. `max_sequence_length` changes the maximum length the model can process, and would possibly even shorten it if set too low. As such, if we give default that is too small, users may accidentally lessen the maximum length. On the flip side, if we give a default that is too large, it might be needlessly wasteful.

I'll make this a required argument and make sure the docstring is appropriately clear!

Reply:
Sounds good!